css/app.css

```diff
@@ -16,6 +16,9 @@ html, body { background: #eee url(../images/background.png); color: #333; cursor
 #social .icon { cursor: pointer; height: 40px; pointer-events: none; width: 40px; }
 #social .icon-label { display: inline; margin: 0; padding: 0; position: relative; top: -15px; }
 
+#links { background: rgba(255, 255, 255, 0.5); padding: 10px 20px 6px 20px; }
+#links .link-text { display: inline; margin: 0; padding: 0; position: relative; }
```

Comment:
@sarahmangels non-blocking nit: since we only have a single anchor type right now, can this just target the `a` and not the specific class?

Reply:
this is why there are no women in stem